.github/workflows/reusable_trivy-scans.yml

```diff
@@ -1,51 +1,39 @@
 ---
 name: Run Trivy Scans
+
 on:
-    workflow_call:
-jobs:
-    build-image:
-        runs-on: ubuntu-latest
-        steps:
-            - name: Check out GitHub Repo
-              uses: actions/checkout@v4
+  workflow_call:
 
-            - name: Build an image from Dockerfile
-              run: |
-                  git config --global --add safe.directory $GITHUB_WORKSPACE;
-                  docker build -t trivy-test .
+jobs:
+  build-and-scan-image:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      security-events: write
 
-            - name: Run Trivy vulnerability scanner, output as table
-              uses: aquasecurity/trivy-action@master
-              with:
-                  image-ref: "trivy-test"
-                  format: "table"
-                  output: "trivy-results.tbl"
-                  timeout: "20m0s"
+    steps:
+      - name: Check out GitHub Repo
+        uses: actions/checkout@v4
 
-            - name: Check for log4j CVEs
-              run: |
-                  set -e
-                  for cve in CVE-2021-4104 CVE-2021-44228 CVE-2021-45046 CVE-2022-22965 ; do
-                  	if grep -c $cve trivy-results.tbl > 0;
-                  	  then
-                  	    echo "Found log4j error: $cve"
-                  	    exit 2
-                  	fi
-                  done
-                  # If above doesn't find errors, return pass message:
-                  echo "Did not find core log4j errors, yay!"
+      - name: Build an image from Dockerfile
+        run: |
+          docker build -t kbase/trivy:scan . 
 
-            - name: Rerun Trivy vulnerability scanner for GitHub security
-              uses: aquasecurity/trivy-action@master
-              if: always()
-              with:
-                  image-ref: "trivy-test"
-                  format: "sarif"
-                  output: "trivy-results.sarif"
-                  timeout: "20m0s"
-                  ignore-unfixed: true
+      - name: Run Trivy vulnerability scanner
+        uses: aquasecurity/trivy-action@master
+        with:
+          image-ref: "kbase/trivy:scan"
+          format: "sarif"
+          output: "trivy-results.sarif"
+          timeout: "20m"
+          ignore-unfixed: true
+          severity: 'CRITICAL'
```

Comment:
I thought we were going to do critical and high?

Reply:
I think we should stick with critical first.

Reply:
Ok, we can come back to it later I guess

Reply:
We could also run a one off trivy scan with the trivy CLI and add those to a spreadsheet.

Reply:
IOW scan the images ourselves and take GHA out of the picture? We could still do that in the cron job I assume